README.md

```diff
@@ -26,35 +27,66 @@ Additional handlers are currently under development.
   - Type is always set to "suspicious"
   - Metadata field contains symbol and price
 
+<!-- -->  
 - AE-AAVE-ADDRESS-WATCH
   - Fired when any address from a pre-defined watchlist is involved in a transaction
   - Severity is always set to "low"
   - Type is always set to "suspicious" 
   - Metadata field contains "from" address and transaction hash
-  
+
+<!-- -->
 - AE-AAVE-ADMIN-EVENT
   - Fired when a transaction log contains an event that matches a list of Aave administrative events
   - Severity is always set to "low"
   - Type is always set to "suspicious" 
   - Metadata field contains Aave contract name and address, event name, and transaction hash
 
+<!-- -->
 - AE-AAVE-HIGH-TX-AMOUNT
   - Fired when an Aave Borrow/Deposit/Repay/Withdraw event crosses a predefined threshold amount
   - Severity is always set to "medium"
   - Type is always set to "suspicious" 
   - Metadata field contains the event log data
 
+<!-- -->
 - AE-AAVE-TVL
-  - Fired when available liquidity, total stable debt, total variable debt, total debt or total value locked drastically changes
+  - Fired when available liquidity, total stable debt, total variable debt, total debt or total
+    value locked drastically changes
   - Severity is always set to "high"
   - Type is always set to "suspicious"
   - Metadata field contains additional alert information (field, reserve, observation, average)
 
+<!-- -->
 - AE-AAVE-PRICE-ORACLE-STALE
   - Fired when data from an AAVE price oracle (Chainlink aggregator) is older than 24 hours
-  - Severity is always set to 'medium'
-  - Type is always set to 'suspicious'
-  - Metadata field contains additional alert information (token symbol, token address, oracle data age, oracle contract address)
+  - Severity is always set to "medium"
+  - Type is always set to "suspicious"
+  - Metadata field contains additional alert information (token symbol, token address, oracle data
+    age, oracle contract address)
+
+<!-- -->
+- AE-AAVE-NEW-CONTRACT-INTERACTION
+  - Fired when a relatively new contract interacts with the Aave Lending Pool contract
```

Comment:
Define `relatively new` either with the default value or with an example.

```suggestion
  - Fired when a relatively new contract (age defined in config JSON file) interacts with the Aave Lending Pool contract
```
or
```suggestion
  - Fired when a relatively new contract (e.g. less than 7 days old) interacts with the Aave Lending Pool contract
```

Reply:
updated using suggestion #1